### PR TITLE
fixed eslint bugs

### PR DESCRIPTION
**Pull request by roman454647**

### CODE REVIEW

File: client/src/components/FormValidator/index.jsx [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+    }, this.createValidObj())
+  }
 
-    createValidObj() {
+  createValidObj() {
     const validation = this.validations.reduce((result, rule) => (
-       {...result,[rule.field]:{ isInvalid: false, message: '' }}
-    ),{});
-    return { isValid: true, ...validation};
+      { ...result, [rule.field]:{ isInvalid: false, message: '' } }
```
Comment (InnaKutsik) [inline client/src/components/FormValidator/index.jsx]: pls use columns

```
{
...result,
[rule.field]:{
     isInvalid: false,
     message: '',
 }
}
```

File: client/src/components/Register/index.jsx [hunk 2]
```diff
@@ -1,146 +1,170 @@
...
-                            <div className="mobile-link">
-                                <Link to="/login" className="mobile-link">Already have an account?</Link>
-                            </div>
-                        </div>
-                    </div>
+    const { register } = this.props
+    register(newUser)
+    const validation = this.validator.validate(state)
+    this.setState((prevState) =>
+      ({ validation: this.validator.validate(prevState), submitted:true })
```
Comment (InnaKutsik) [inline client/src/components/Register/index.jsx]: you rewrite all another params
```

this.setState((prevState) => ({
....prevState,
validation: this.validator.validate(prevState),
submitted:true
});
```

Comment (roman454647) [reply client/src/components/Register/index.jsx]: done

File: client/src/components/FormValidator/index.jsx (hunk 1 above)
Comment (roman454647) [reply client/src/components/FormValidator/index.jsx]: done

**Pull request merged**